api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -74,10 +97,33 @@ type VolumeReplicationGroupSpec struct {
 	S3SecretName string `json:"s3SecretName,omitempty"`
 }
 
+type ProtectedPVC struct {
+	// Name of the VolRep resource
+	Name string `json:"name,omitempty"`
+
+	// Conditions for each protected pvc
+	Conditions []metav1.Condition `json:"conditions,omitempty"`
+}
+
+type ProtectedPVCMap map[string]*ProtectedPVC
```

Comment:
Nope actually. But since both the names of ProtectedPVC and ProtectedPVCs are already used (for structure type and for conditions array), this name was used. Unless you think the name has to be changed now, I think this can be done later.

Reply:
I was actually thinking we do not need the type as well, and just use a map in the status. But can be taken up subsequently as well.